test/unit/lib/plugins/aws/remove/lib/bucket.test.js

```diff
@@ -37,17 +37,17 @@ describe('emptyS3Bucket', () => {
     });
   });
 
-  describe('#listObjects()', () => {
-    it('should resolve if no objects are present', () => {
-      const listObjectsStub = sinon.stub(awsRemove.provider, 'request').resolves();
+  describe('#listObjectVersions()', () => {
```

Comment:
We've established the approach where new functionality requires modification to tests written the old way (without `runServerless`), they should be replaced with modern `runServerless`-based version to not increase technical debt as we're slowly moving away from these old, fragile tests. Would you be able to replace these with `runServerless`-based equivalent?